make_profiler/report/script.js

```diff
@@ -0,0 +1,157 @@
+function filterTarget() {
+    let input, filter, table, tr, td, i, txtValue;
+    input = document.getElementById("myInput");
+
+    filter = input.value.toUpperCase().split(" ").join("_");
+    table = document.getElementById("statusTable");
+    tr = table.getElementsByTagName("tr");
+
+    if (filter.startsWith('"') && filter.endsWith('"')) {
+        filter = filter.substring(1, filter.length - 1)
+        for (i = 0; i < tr.length; i++) {
+            td = tr[i].getElementsByTagName("td")[0];
+            if (td) {
+                txtValue = td.childNodes[0].textContent || td.innerText;
+                if (txtValue.toUpperCase() === filter) {
+                    tr[i].style.display = "";
+                } else {
+                    tr[i].style.display = "none";
+                }
+            }
+        }
+    } else {
+        for (i = 0; i < tr.length; i++) {
+            td = tr[i].getElementsByTagName("td")[0];
+            if (td) {
+                txtValue = td.textContent || td.innerText;
+                if (txtValue.toUpperCase().includes(filter)) {
+                    tr[i].style.display = "";
+                } else {
+                    tr[i].style.display = "none";
+                }
+            }
+        }
+    }
+}
+
+//refresh page every 5 minutes
+setInterval(function () {
+    getStatus(url);
+}, 1000 * 5 * 60);
+
+const url = "report.json";
+const errorTxt = "Either not using a web server or can not reach status report details!"
```

Comment:
Can this be phrased positively so that it is clear what to do to make the thing work?

Reply:
Can not reach status report.json file! Please check pipeline and check related file on server